internal/store/interface.go

```diff
@@ -49,6 +49,13 @@ type Repository interface {
 	// @serverID: required
 	// @conditionKind: required
 	Delete(ctx context.Context, serverID uuid.UUID, conditionKind rctypes.Kind) error
+
+	// AddServer creates a server record in FleetDB
+	// @serverID: required
+	// @bmcAddr: required
+	// @bmcUser: required
+	// @bmcPass: required
+	AddServer(ctx context.Context, serverID uuid.UUID, bmcAddr string, bmcUser string, bmcPass string) error
```

Comment:
We might want a different interface here for managing the server population. Conditions are moving to a NATS backed system and I am (was?) intending on deleting the Server-service condition store implementation.

Reply:
I am not familiar with the future plan about refactoring of the `Conditionorc` repo thus may need some insights how we want to scope the new interface.
As a temporary workaround, I am making a public function which receives the `*ServerService` store as a parameter.

Reply:
In short, I had been planning on removing `internal/store/serverservice.go`. That said, we might want to provide a nice interface to FleetDB to get various database objects. We could create a `Servers` interface for FleetDB that encapsulated the client and provided an API that was less "raw" for manipulating objects in the Servers table (and associated attributes).